.drone.yml

```diff
@@ -184,6 +186,28 @@ steps:
     instance:
     - drone-publish.rancher.io
 
+- name: docker-publish
+  image: plugins/docker
+  settings:
+    build_args:
+      - ARCH=amd64
+      - TAG=harvester-${DRONE_TAG}-amd64
+    context: /drone/harvester-installer/package
+    custom_dns: 1.1.1.1
+    dockerfile: /drone/harvester-installer/package/Dockerfile
+    password:
+      from_secret: docker_password
+    repo: "rancher/harvester-bundle"
+    tag: "${DRONE_TAG}"
```

Comment:
not sure if we need to submit the EIO ticket in order to have a new repo of the bundle images.

Reply:
Recorded in https://github.com/rancherlabs/eio/issues/389